File: libzxing/src/main/java/com/google/zxing/ScanCallback.java
package com.google.zxing;

import android.graphics.Bitmap;

/**
 * 二维码扫描工具类
 */
public class ScanCallback {

    public static final String RESULT_TYPE = "result_type";
    public static final String RESULT_STRING = "result_string";
    public static final int RESULT_SUCCESS = 1;
    public static final int RESULT_FAILED = 2;

    /**
     * 解析二维码结果的回调，可以自定义扫描成功后行为
     */
    public interface AnalyzeCallback{

        public void onAnalyzeSuccess(Bitmap mBitmap, String result);

        public void onAnalyzeFailed();
    }
}

File: libzxing/src/main/java/com/google/zxing/ScanCallback.kt
package com.google.zxing

import android.graphics.Bitmap

/**
 * 二维码扫描工具类
 */
object ScanCallback {
    const val RESULT_TYPE: String = "result_type"
    const val RESULT_STRING: String = "result_string"
    const val RESULT_SUCCESS: Int = 1
    const val RESULT_FAILED: Int = 2

    /**
     * 解析二维码结果的回调，可以自定义扫描成功后行为
     */
    interface AnalyzeCallback {
        fun onAnalyzeSuccess(mBitmap: Bitmap?, result: String?)

        fun onAnalyzeFailed()
    }
}
